code/modules/food_and_drinks/food/snacks_soup.dm

```diff
@@ -225,3 +225,16 @@
 	bonus_reagents = list("nutriment" = 4, "vitamin" = 6)
 	tastes = list("beet" = 1)
 	foodtype = VEGETABLES
+
+/obj/item/reagent_containers/food/snacks/soup/monkey
+	name = "sopa de macaco"
+	desc = "A space-Brazilian delicacy. Uma delicia!"
+	icon_state = "sopademacaco"
+	bonus_reagents = list("nutriment" = 5, "vitamin" = 10)
+	tastes = list("delicia" = 1)
+	foodtype = MEAT
+
+/obj/item/reagent_containers/food/snacks/soup/monkey/attack(mob/M, mob/user, def_zone)
+	if(..())
+		for(var/mob/living/carbon/human/H in oview(M))
```

Comment:
add a cooldown of some description, add some kind of feedback message and consider the quality of your changes before making them in future

Reply:
a cooldown to what bro

Reply:
if you mean adding a cooldown to the disgust being added which means you cant eat 5 sopa de macaco bowls and gross someone out severely you need to posterize yourself nigga you dont know how this works....